bokeh/io/export.py

```diff
@@ -24,35 +24,37 @@
 import warnings
 from os.path import abspath
 from tempfile import mkstemp
+from typing import Any, List, Optional, Tuple, Union, cast
 
 # External imports
 from PIL import Image
 
 # Bokeh imports
+from ..document import Document
 from ..embed import file_html
-from ..resources import INLINE_LEGACY
+from ..models.layouts import LayoutDOM
+from ..resources import INLINE, Resources
 from .util import default_filename
+from .webdriver import WebDriver, webdriver_control
```

Comment:
@mattpap This import prevents using Bokeh without having geckodriver/chromedriver installed:
```
>>> from bokeh.plotting import figure
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/p-himik/dev/git/bokeh/bokeh/plotting/__init__.py", line 72, in <module>
    from ..io import curdoc; curdoc
  File "/home/p-himik/dev/git/bokeh/bokeh/io/__init__.py", line 23, in <module>
    from .export import export_png, export_svgs
  File "/home/p-himik/dev/git/bokeh/bokeh/io/export.py", line 38, in <module>
    from .webdriver import WebDriver, webdriver_control
  File "/home/p-himik/dev/git/bokeh/bokeh/io/webdriver.py", line 163, in <module>
    webdriver_control = _WebdriverState()
  File "/home/p-himik/dev/git/bokeh/bokeh/io/webdriver.py", line 115, in __init__
    raise RuntimeError("Neither firefox/geckodriver nor chromium-browser/chromedriver are available on system PATH. " \
RuntimeError: Neither firefox/geckodriver nor chromium-browser/chromedriver are available on system PATH. You can install the former with 'conda install -c conda-forge firefox geckodriver'.
```

Reply:
@p-himik, this should be fixed in PR #9670.